pipelines/core-services/slack-notification.yaml

```diff
@@ -0,0 +1,24 @@
+- op: add
+  path: /spec/params/-
+  value:
+    name: slack-webhook-notification-team
+    default: ""
+- op: add
+  path: /spec/finally/-
+  value:
+    name: slack-webhook-notification
+    taskRef:
+      name: slack-webhook-notification
+      version: "0.1"
+    when:
+    - input: $(params.slack-webhook-notification-team)
+      operator: notin
+      values: [""]
+    - input: $(tasks.status)
+      operator: in
+      values: ["Failed"]
+    params:
+    - name: message
+      value: Tekton pipelineRun $(context.pipelineRun.name) failed
```

Comment:
would it make sense to add more useful information, like the namespace where the pipelinerun failed? or would this be enough for people to know where to look when they got this message

Reply:
that's tricky since we do not have much information inside the pipeline, I can add namespace but hardcoding url is probably not right.